cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/internal/query/IndexTest.java

```diff
@@ -19,6 +19,7 @@
 import static org.hamcrest.Matchers.notNullValue;
 import static org.hamcrest.Matchers.nullValue;
 
+import com.cloudant.sync.internal.query.callables.CreateIndexCallable;
```

Comment:
Remove import now that you've moved tests to new `CreateIndexCallableTests` class.